packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -2,7 +2,10 @@
   <header class="hidden" id="fxa-connect-another-device-header"><h1>{{#t}}Connect another device{{/t}}</h1></header>
   {{#showSuccessMessage}}
     {{#isSignedIn}}
-      <div class="success success-authenticated visible">{{#t}}This Firefox is connected{{/t}}</div>
+      <div id="fxa-connected" class="flex flex-nowrap justify-center">
+        <div id="circle-check" class="bg-[#B3FFE3] rounded-[50%] shrink-0 h-8 w-8 mt-3.5 relative"></div>
```

Comment:
Q: Why the custom bg-color?
Also, re: `rounded-[50%]` class, what about something like `rounded-full`?
https://tailwindcss.com/docs/border-radius#rounded-corners

Reply:
Re: `mt-3.5`, any chance we want to round that to 3 or 4? I think this was the only instance of a fractional margin I've seen in the codebase so far, per <kbd>git grep -Eoh "\bm[tblr]\-[0-9.]+" | sort | uniq</kbd>.

Reply:
> Q: Why the custom bg-color? 

It's from the Figma design linked from the issue.  Sorry I don't have a full answer.  Maybe Vesta know who did the design.

> Also, re: `rounded-[50%]` class, what about something like `rounded-full`? https://tailwindcss.com/docs/border-radius#rounded-corners

First thing I tried.  It did not work.  I think the actual CSS for that was border-radius: 999px or something like that.

Reply:
> Re: `mt-3.5`, any chance we want to round that to 3 or 4? I think this was the only instance of a fractional margin I've seen in the codebase so far, per git grep -Eoh "\bm[tblr]-[0-9.]+" | sort | uniq.

Since it's something defined in Tailwind I don't understand what the downside is with using it.  I'm no Tailwind expert though.